protos/feast/core/StreamFeatureView.proto

```diff
@@ -95,4 +96,7 @@ message StreamFeatureViewMeta {
 
     // Time where this Feature View is last updated
     google.protobuf.Timestamp last_updated_timestamp = 2;
+
+    // List of pairs (start_time, end_time) for which this feature view has been materialized.
+    repeated MaterializationInterval materialization_intervals = 3;
```

Comment:
hm do we really need this? I don't think we're planning on materializing SFVs

Reply:
actually am fine with this for now, ideally will remove it long term

Reply:
addressed. We don't materialize but since streamfeatureview inherits featureview, will need this param in proto.

Reply:
Technically you can still run batch materialization for a stream feature view right?